database/cassSeed.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const faker = require('faker');
+const { argv } = require('yargs');
+
+const start = argv.start || 1;
+const finish = argv.finish || 2;
+
+const filename1 = argv.output1 || 'cassPhotosByRoom.csv';
+const filename2 = argv.output2 || 'cassListByUser.csv';
+
+const stream1 = fs.createWriteStream(filename1);
+const stream2 = fs.createWriteStream(filename2);
+
+const photosByRoom = (roomId) => {
+  const randomNumPhotos = Math.floor(Math.random() * 17);
+  const room_id = roomId;
+  let str = '';
+
+  for (let i = 1; i < randomNumPhotos; i += 1) {
+    const photo_order = i;
+    const title = faker.lorem.words();
+    const address = faker.address.streetAddress();
+    const image_url = faker.image.imageUrl();
+    const image_description = faker.lorem.words();
+
+    str += `${room_id},${photo_order},${title},${address},${image_url},${image_description}\n`;
+  }
+
+  return `${str}`;
+};
+
+let listId = 1;
+
+const listByUser = (userId) => {
+  const randomNumList = Math.floor(Math.random() * 5);
+  const randomNumRoom = Math.floor(Math.random() * 8);
+  const user_id = userId;
+  let str = '';
+  for (let j = 1; j < randomNumList; j += 1) {
+    const list_name = faker.lorem.words();
+
+    for (let k = 1; k < randomNumRoom; k += 1) {
+      const list_id = listId;
+      listId += 1;
+      const room_id = Math.floor(Math.random() * 10000000) + 1;
+      str += `${user_id},${list_id},${list_name},${room_id}\n`;
+    }
+  }
+  return `${str}`;
+};
+
+const startWriting = (writeStream, encoding, dataFunction, done) => {
```

Comment:
not sure if you need to mention "encoding" cause I think it is utf-8 by default, but I can be wrong